### ISSUE TITLE
Use of ONIX code list 196 - codes 08 and 09 - Unknown accessibility / not accessible

### ISSUE THREAD

**Issue by chrisONIX** [w3c/publ-a11y#271]
In ONIX code list 196 - there are two codes that can be used by a publisher / creator to indicate in the case of code 08 - Unknown accessibility. In the case of 09 - Inaccessible, or known limited accessibility.
Both of these codes can be accompanied by a short text to explain.
In the case of 08 - the feedback from publishers is that they will use this code when a title is not yet assessed or they do not yet have the information - but these titles may already have been announced in the supply chain. In the display guidelines should we mention these two cases.  In the case of 08 -  Unknown accessibility - this is very much an example of where a publisher will later send an updated ONIX file with accessibility information.
There is an intention to particularly use this for older ePub titles awaiting evaluation and also for PDFs.

How should we approach these cases in our guidelines?

**Comment by GeorgeKerscher** [comment]
It seems that it might be best to place a note at the end of section 1.2 just after the reference to ONIX techniques. The note might read:

Publishers update their ONIX records  as needed. We expect unknown accessibility metadata will be provided as more information becomes known. For this reason, implementors should be prepared to update the accessibility metadata as new ONIX feeds are made available.

**Comment by chrisONIX** [comment]
In the ONIX display techniques document in the OTHER section we can give advice about displaying the information about 

ONIX code list 196 - code 08 - Unknown accessibility

This value will be used when a product has not yet been assessed or reassessed for accessibility. It may be a forthcoming title. The information that the information is not yet known should be displayed when there is a positive indication of this in the metadata ?

ONIX code list 196 - code 09 - Inaccessible, or known limited accessibility

this will be used when products are of very limited accessibility and this code would be used on its own in ONIX. This information should be displayed when available and when there is a positive indication in the metadata that the title is not accessible.
